src/components/content-header-card/index.tsx

```diff
@@ -0,0 +1,178 @@
+import classNames from 'classnames'
+import BadgeList, { Badge } from 'components/badge-list'
+import Button from 'components/button'
+import Card from 'components/card'
+import DropdownDisclosure, {
+	DropdownDisclosureLinkItem,
+} from 'components/dropdown-disclosure'
+import Heading from 'components/heading'
+import IconTileLogo from 'components/icon-tile-logo'
+import StandaloneLink from 'components/standalone-link'
+import Text from 'components/text'
+import { ProductSlug } from 'types/products'
+import s from './content-header-card.module.css'
+
+interface DropdownItem {
+	text: string
+	href: string
+}
+
+interface Dropdown {
+	text: string
+	items: Array<DropdownItem>
+}
+
+interface Link {
+	text: string
+	href: string
+	icon?: React.ReactElement
+}
+
+interface Button {
+	text: string
+	icon?: JSX.IntrinsicElements['svg']
+	isPrimary?: boolean
+	onClick: () => void
+}
+
+export interface ContentHeaderCardProps {
+	className?: string
+	icon?: Exclude<ProductSlug, 'sentinel'>
+	title: string
+	attribution?: string
+	description?: string
+	note?: string
+	badges?: Array<Badge>
+	dropdown?: Dropdown
+	links?: Array<Link>
+	buttons?: Array<Button>
+}
+
+export default function ContentHeaderCard({
+	className,
+	icon,
+	title,
+	attribution,
+	description,
+	note,
+	badges,
+	dropdown,
+	links,
+	buttons,
+}: ContentHeaderCardProps) {
+	return (
+		<Card
+			elevation="base"
+			className={classNames(s.contentHeaderCard, className)}
+		>
+			<div className={s.cardTop}>
+				{icon && (
+					<IconTileLogo className={s.icon} size="large" productSlug={icon} />
+				)}
+				<div className={s.content}>
+					<div className={s.upper}>
```

Comment:
I think it could be nice for this upper and lower sections to render a slot or component instead of being props driven, given the amount of conditional rendering. Or if we want to keep it opinionated, perhaps not elevating it as a global component yet until we have examples of other use-cases.